cmd/broker/fanout/main.go

```diff
@@ -99,7 +99,7 @@ func main() {
 	if err != nil {
 		logger.Fatal("Failed to create fanout sync pool", zap.Error(err))
 	}
-	if _, err := handler.StartSyncPool(ctx, syncPool, syncSignal, env.MaxStaleDuration, handler.DefaultHealthCheckPort); err != nil {
+	if _, err := handler.StartSyncPool(ctx, syncPool, syncSignal, env.MaxStaleDuration, handler.DefaultReadinessAndLivenessCheckPort); err != nil {
```

Comment:
I feel it's a bit verbose to say "ReadinessAndLiveness". How about just ProbeCheck?

Reply:
That's a fantastic suggestion! I like the name. will change to it